client/src/utils/socket-context.tsx

```diff
@@ -54,54 +45,59 @@ export const SocketGateway = ({ children }: PropsWithChildren<any>) => {
 	const [logs, setLogs] = useState<string[]>([]);
 	const [aprsPacket, aprsDispatch] = useReducer(aprsReducer, {} as IPacket);
 	const [loRaPacket, loraDispatch] = useReducer(loraReducer, {} as IPacket);
-	const socket = io('http://localhost:8086/toClient');
+	const [packetFrequency, setPacketFrequency] = useState<number>(3);
+	const [protocol, setProtocol] = useState<string>('APRS');
```

Comment:
```suggestion
	const [packetFrequency, setPacketFrequency] = useState<number>(3);
	const [protocol, setProtocol] = useState<string>('APRS');
```
Why are these the default values?

Reply:
Protocol is either 'APRS' or 'LORA'. It is my understanding LORA is being phased out, so APRS is used here as the default. packetFrequency is the time between packets in streaming mode. So when it is set to a default of 2, that means there will be 2 seconds between packets. Both of these can be changed using socketContext hook